crates/router/tests/connectors/adyen.rs

```diff
@@ -358,27 +358,6 @@ async fn should_fail_payment_for_incorrect_card_number() {
     );
 }
 
-// Creates a payment with empty card number.
```

Comment:
Please remove this test from the connector template as well:

https://github.com/juspay/hyperswitch/blob/17e8305a5de49a37aca9205f4dcdc8765c7cd0ec/connector-template/test.rs#L305-L326